samples/config_helper_vulkan.cc

```diff
@@ -46,6 +46,7 @@ const size_t kNumberOfRequiredValidationLayers =
 const char kVariablePointers[] = "VariablePointerFeatures.variablePointers";
 const char kVariablePointersStorageBuffer[] =
     "VariablePointerFeatures.variablePointersStorageBuffer";
+const char kFloat16Int8[] = "Float16Int8Features.shaderFloat16";
```

Comment:
This should probably be kFloat16Int8Float16 or something equally ridiculous if you want to eventually support the int8 feature bit.

Reply:
Done.